### PR TITLE
 Add flag to render in binary/decimal suffix in DownloadColumn

### PR DESCRIPTION
**Pull request by amartya-dev**
Add a test for rendering binary suffix in test_progress.py

## Type of changes

- [ ] Bug fix
- [x] New feature
- [ ] Documentation / docstrings
- [x] Tests
- [ ] Other

## Checklist

- [x] I've run the latest [black](https://github.com/psf/black) with default args on new code.
- [x] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [x] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

Please describe your changes here. If this fixes a bug, please link to the issue, if possible.
An additional feature in #323

### CODE REVIEW

**Comment by amartya-dev** [comment]
@willmcgugan I have added the discussed feature, let me know in case anything else is required

**Review by willmcgugan — CHANGES_REQUESTED** [review]
Rather than "decimal_suffix", let's call it "binary_units" which should default to False.

File: rich/progress.py [hunk 1]
```diff
@@ -292,15 +291,30 @@ def render(self, task: "Task") -> Text:
...
 class DownloadColumn(ProgressColumn):
-    """Renders file size downloaded and total, e.g. '0.5/2.3 GB'."""
+    """Renders file size downloaded and total, e.g. '0.5/2.3 GB'.
+
+    Args:
+        decimal_suffix (bool, optional): Flag to render filesize in decimal format, disable to render in binary. Defaults to True.
+    """
+
+    def __init__(self, decimal_suffix: bool = True) -> None:
+        self.decimal_ssuffix = decimal_suffix
```
[the remark this answers is not in the document]
Comment (amartya-dev) [reply rich/progress.py:301]: Alright!!

**Review by Edward-Knight — APPROVED** [review]

**Comment by willmcgugan** [comment]
LGTM. Thanks.

**Pull request merged**